package.json

```diff
@@ -26,17 +26,21 @@
   "dependencies": {
     "axe-core": "3.1.2",
     "jest-matcher-utils": "24.0.0",
-    "lodash.merge": "4.6.1"
+    "lodash.merge": "^4.6.2"
   },
   "devDependencies": {
     "async-to-gen": "1.4.0",
     "jest": "24.1.0",
     "react": "16.7.0",
-    "react-dom": "16.7.0"
+    "react-dom": "16.7.0",
+    "@testing-library/react": "^8.0.5"
   },
   "jest": {
     "transform": {
       "^.+\\.js$": "async-to-gen"
-    }
+    },
+    "transformIgnorePatterns": [
+      "node_modules/(?!(@testing-library/react)/)"
```

Comment:
This was the offending line @citypaul. I forgot to update the ignore pattern to coincide with the new naming convention for react testing library:
`react-testing-library` became `@testing-library/react`

Reply:
Ah, this is fantastic! Thank you for this. There must be many others experiencing the same issue, so this is great!